### ISSUE TITLE
Fixed toolbar

### ISSUE THREAD

**Issue by Caesar-ch** [codex-team/editor.js#2370]
How to fix the toolbar instead of clicking the plus sign before the toolbar appears

**Comment by sbesic-starke** [comment]
Would also like to know... Seems like a pretty basic feature that's required for any good text editor to have, or the documentation is just lacking it once again.

**Comment by Caesar-ch** [comment]
I don't see that in the documentation

**Comment by miketalley** [comment]
I don't think this is currently available. I would say this is not a bug and more likely a feature request.

Also, see this discussion: https://github.com/codex-team/editor.js/discussions/1943
